post-docs/add-back-references.py

```diff
@@ -40,11 +41,14 @@ class GenerationType(enum.Enum):
 
 
 def download_file(url):
-    filedata = urlopen(url)
-    datatowrite = filedata.read()
+    try:
+        filedata = urlopen(url)
+        datatowrite = filedata.read()
 
-    with open('redirects.txt', 'wb') as f:
-        f.write(datatowrite)
+        with open('redirects.txt', 'wb') as f:
+            f.write(datatowrite)
+    except URLError as e:
+        log.warning(e)
```

Comment:
Should not we also skip generation in this case?

Reply:
(I think what is going to happen with this version is that will keep on using the redirects.txt retrieved from previous successfully downloaded redirect.txt ? I guess it would be great to  do two things:

1) make sure that the redirects.txt is named differently depending on a provider  (just in case of programming mistakes) 

2) store them somewhere where they will be deleted (tmp dir)

3) if download fails, skip the whole `generate`

Reply:
Very good point. Will handle these points soon and re request for review :)